app/views/hyrax/users/index.html.erb

```diff
@@ -1,21 +1,50 @@
-<div id="search">
-  <%= render 'search_form' %>
-  <h1><%= application_name %> Users</h1>
-</div>
-<table class="table table-striped">
-    <thead>
-        <tr>
-            <th>Avatar</th>
-            <th class="sorts"><i id="name" class="<%= params[:sort].blank? ? 'caret up' : params[:sort] == "name desc" ? 'caret' : params[:sort] == "name" ? 'caret up' : '' %>"></i> User Name</th>
-            <th class="sorts"><i id="login" class="<%= params[:sort] == "login desc" ? 'caret' : params[:sort] == "login" ? 'caret up' : '' %>"></i> User Id</th>
-            <th class="sorts"><i id="department" class="<%= params[:sort] == "department desc" ? 'caret' : params[:sort] == "department" ? 'caret up' : ''%>"></i> Department</th>
-            <th>Works Created</th>
-        </tr>
-    </thead>
-    <tbody>
-    <%= render partial: 'user_row', collection: @users, as: :user %>
-    </tbody>
-</table>
-<div class="pager">
-  <%= paginate @users, theme: 'blacklight', route_set: hyrax %>
+<% content_for :page_header do %>
+  <h1><span class="fa fa-user"></span> <%= t('hyrax.users.index.title') %></h1>
+<% end %>
+
+<div class="panel panel-default users-listing">
+  <div class="panel-heading">
+    <% count_users = @presenter.user_count %>
+    <% admin_count = @presenter.repository_admin_count %>
+    <% if count_users == 1 %>
+      <%= t('hyrax.users.index.describe_users_singular_html', nbr_admin: admin_count) %>
+    <% elsif admin_count == 1 %>
+      <%= t('hyrax.users.index.describe_users_multiple_one_admin_html', nbr_users: count_users) %>
+    <% else %>
+      <%= t('hyrax.users.index.describe_users_multiple_html', nbr_users: count_users, nbr_admin: admin_count) %>
+    <% end %>
+  </div>
+
+  <div class="panel-body">
+    <div class="table-responsive">
+      <table class="table table-striped datatable">
+        <thead>
+          <tr>
+            <th></th>
+            <th><%= t('.id_label') %></th>
+            <th><%= t('.role_label') %></th>
+            <th><%= t('.access_label') %></th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @presenter.users.each do |user| %>
+            <tr>
+              <td><%= link_to hyrax.profile_path(user) do %>
+                    <%= image_tag(user.avatar.url(:thumb), width: 30) if user.avatar.file %>
+                  <% end %>
+              </td>
+              <td><%= link_to user.email, hyrax.profile_path(user) %></td>
+              <% @presenter.user_roles(user).each do |role| %>
+                <td><%= role %></td>
```

Comment:
Possibly... I intended it to be separate lines in the row! Though currently the ONLY possible role is 'admin' so I probably didn't test this.